mentor-guidance.md

```diff
@@ -0,0 +1,64 @@
+# Mentor Guide
+
+## Getting Started
+
+If IOOS is included as a GSoC mentoring organization for the year (typically announced in late February per the [GSoC program timeline](https://developers.google.com/open-source/gsoc/timeline), we invite interested mentors to propose project ideas using our [GSoC Project Issue Template](https://github.com/ioos/gsoc/issues/new?template=gsoc-project-proposal.yml). 
+
+The process for GSoC follows this rough outline:
+1. **Mentor** submits a project idea to the IOOS Organization following the guidance here.
+1. IOOS reviews the list of project ideas to determine how many slots to request.
+1. IOOS submits an application to Google with the list of proposed project ideas.
+1. Google reviews the organization application and notifies which ogranizations have been accepted and what number of slots the organization has been granted.
+1. If accepted, potential GSoC contributors will engage with the project ideas via GitHub issues.
+1. **Mentors** collaborate with potential contributors and evaluate their suitability for the project. See [evaluating contributors](#evaluating-project-contributors).
+1. Contributors submit applications to IOOS.
+1. IOOS ranks applications and submits them to Google.
+1. Google announces accepted projects.
+1. If accepted, coding runs for four months (typically May - August). **Mentors** support the project throughout the coding period.
+1. Contributors submit mid-term and final summaries of their work.
+1. Mid-term and final evaluations are submitted by the Mentor. These are breakpoints at which a project can be failed. See [Warning Signs](https://google.github.io/gsocguides/mentor/warning-signs) for more details on warning signs to look out for in a project.
+
+**Further Background:** 
+* [Google Summer of Code Guide - Mentor](https://google.github.io/gsocguides/mentor/) has great information about being a GSoC mentor.
+
+### Do I have the experience to be a mentor?
+
+The answer is generally: **Yes**.
+Note that our organization values creativity, intelligence and enthusiasm above specific knowledge of the libraries or algorithms we use.
```

Comment:
Minor comment: This phrase seems to be target at participants and not mentors. While those characteristics are good in a mentor as well, we do hope that they have enough knowledge on their library to be able to help and mentor the  participant.